go/chat/boxer.go

```diff
@@ -1069,6 +1069,11 @@ func (b *Boxer) compareHeadersMBV2orV3(ctx context.Context, hServer chat1.Messag
 		return NewPermanentUnboxingError(NewHeaderMismatchError("EphemeralMetadata"))
 	}
 
+	// BotUID (only present in V3 and greater)
```

Comment:
since key reader support and chat reader support are going out in the same release i didn't bump this version